src/pages/support/faq.md

```diff
@@ -26,13 +27,35 @@ When retrieving data from some external systems your code might need to connect
 
 If you have a need for a port that is not in this list, please share with us the use case.
 
+## How can I see usgae reports for my Runtime namespace?
+Runtime currently does not offer usage reports but it is planned to be available in the future. To obtain usage reports for your namespace, kindly reach out to your account manager.
+
+To request usage reports for your namespace, please follow these steps:
+1. Contact your account manager and request for the usage report.
+2. Provide your namespace details to the account manager.
+3. The account manager will provide you with the usage report once it becomes available.
+
+Note: The availability of usage reports is currently not offered by Runtime but it is in the product roadmap. We appreciate your patience and understanding as we continue to improve our services.
+
+## How can I get access to the I/O Runtime activation logs?
+I/O Runtime doesn’t offer a configuration to send activations and logs to an external system, something like Splunk, Datadog or New Relic. This is something we are considering to offer in the future.
+
+Although there is no out-of-the-box integration, there are still ways you can push data from I/O Runtime to an external tool in order to monitor and debug your actions. See [Logging & Monitoring] (https://developer.adobe.com/runtime/docs/guides/using/logging_monitoring/#3rd-party-tools)
+
+## While executing actions, continue to receive ``` "error": "Too many requests in the last minute (count: 73, allowed: 72)."}```  with status code 429. What should I do?
+There is a default throttling limit of 600 activations per minute and 100 concurrent activations for every namespace. These values can be increased per request. To request increase the namespace limits, please follow these steps:
+
+1. Loga a support ticket [here](https://experienceleague.adobe.com/?lang=en&support-tab=home#support).
+2. Provide your namespace and needed limits.
```

Comment:
Can we add a section providing details to customers on how they need to calculate their limits? As currently, `Too many requests at the last minute (count: 73, allowed: 72)` doesn't represent the exact number of requests being throttled.